packages/devtools-reps/src/object-inspector/component.js

```diff
@@ -289,12 +332,30 @@ class ObjectInspector extends Component<Props> {
         repProps.mode = MODE.TINY;
       }
 
-      if (nodeIsLongString(item)) {
+      if (
+        nodeIsLongString(item) ||
+        (evaluation && isLongString(evaluation.getterValue))
+      ) {
         repProps.member = {
-          open: nodeHasFullText(item) && expanded
+          open:
+            expanded &&
+            (nodeHasFullText(item) || nodeHasFullText({ contents: evaluation }))
         };
       }
 
+      if (nodeHasGetter(item)) {
+        const parentNode = getParent(item);
+        const parentGripNode = parentNode && getClosestGripNode(parentNode);
+        const parentGrip = parentGripNode && getValue(parentGripNode);
+        if (parentGrip) {
+          Object.assign(repProps, {
+            onInvokeGetterButtonClick: () =>
+              this.props.invokeGetter(item, parentGrip, item.name),
+            evaluation
+          });
+        }
+      }
```

Comment:
a couple thoughts:

1. is there a way, is it preferable to always have this prop, but only call it when an element is clicked?
2. could we move all of the parent node work into the click handler so the logic is in one place?